### PR TITLE
MOTECH-2644: Update documentation to reflect MOTECH supports IE11+

### PR DESCRIPTION
**Pull request by afijal**

### CODE REVIEW

File: src/common/input-map/input-map.scss [hunk 1]
```diff
@@ -12,8 +12,6 @@
     padding-left: 0px;
     >*{
         display: inline-block;
-    }
-    label, input{
-        width: $sidebar-width;
+        width: 48.2%;
```
Comment (nickdotreid) [inline src/common/input-map/input-map.scss]: "48.2%" is a pretty magic number. Could we avoid this and use flexbox to control the size and layout?

File: src/common/nav.scss [hunk 2]
```diff
@@ -40,7 +40,6 @@ nav {
     nav {
         #{$navItemSelectors}{
             display: block;
-            float: left;
```
Comment (nickdotreid) [inline src/common/nav.scss]: What was this effecting?

**Comment by nickdotreid** [comment]
@afijal ~ I see that you have two commits in this one PR — next time can we try to keep them separate? I think MOTECH-2644 is great, and I think we need to merge both these issues together

-- and I have some edits on MOTECH-2740....

Feel free to push back if you think something would be ugly or take too long

(A) Can the delete button and add buttons line up?

(B) The padding between input elements is a little weird. Currently the label for "value" is closer to the "key" input. There should be more space between the key input and the "value" label

(C) Can we try something else with the existing key-value pairs? Can we make the input element fill the width after the key name?

Here are the issues flagged
motech
 
Here is a quick mock up -- I used `display:flex` and `flex-grow:1` to make items scale to the right size (because flexbox is rad)
image

**Comment by afijal** [comment]
@nickdotreid I had to somehow mess up these commits, lets keep them separated, so I reverted unneccesary changes here.

**Comment by sebbrudzinski** [comment]
test this please

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Pull request merged**